.github/workflows/ci.yml

```diff
@@ -38,7 +38,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        ghc: ["8.10.7", "9.6.6", "9.10.1", "9.12"]
+        ghc: ["9.6.6", "9.10.1", "9.12"]
```

Comment:
We could bump 9.6 to 9.6.7 and 9.12 to 9.12.1. But it doesn't need to be now.

Reply:
Will do that in a follow-up PR where I also update haskell.nix :+1: 

(also, there even are 9.10.2 and 9.12.2 already)